CHANGELOG.rst

```diff
@@ -10,6 +10,9 @@ The third digit is only for regressions.
 Backward-incompatible changes:
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+- Removed the deprecated ``OpenSSL.crypto.CRL``, ``OpenSSL.crypto.Revoked``, ``OpenSSL.crypto.dump_crl``, and ``OpenSSL.crypto.load_crl``.
+- Removed the deprecated ``OpenSSL.crypto.sign`` and ``OpenSSL.crypto.verify``.
```

Comment:
I think we should point users at the cryptography replacement for these APIs in the changelog. However, I'm not even sure what sign did? 😬

Reply:
sign was an RSA PKCS1v15 or ECDSA signature.